index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Responsive Made Easy </title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Oswald&display=swap" rel="stylesheet">
+</head>
+<body>
+    
+    <header class="bg-light text-center">
+        <div class="container--narrow">
+        <p>Lorem, ipsum dolor.</p>
+        <h1>Lorem ipsum dolor sit amet consectetur adipisicing.size is bigger tha</h1>
+        </div>
+    </header>
+    <section>
+        <div class="container">
+        <h2 class="text-center">Quality Designs</h2>
+        <div class="split">
+            <div>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Autem reprehenderit eligendi nemo rerum aliquid sunt ut similique incidunt, aliquam, officiis blanditiis quam. At, ipsam ex.</p>
+            </div>
+            <div>
+                <p>Amet eum doloremque consequatur aliquam dolores nisi est rerum sint ipsam dolorem ipsum illum quas tempora consectetur cumque hic debitis nesciunt molestias incidunt, perspiciatis autem.</p>
+            </div>
+        </div>
+        </div>
+    </section>
+    <section class="bg-dark">
+        <div class="container">
+        <div class="split">
+            <div>
+               <h2>Custom Made Easy for you</h2>
+               <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore odit, dignissimos eius, repellendus quod similique ducimus nesciunt ipsa rem natus molestiae delectus doloribus quasi laudantium.</p>
+            </div>
+            <div>
+                <img src="" alt="man standing on the clif of mountain">
+            </div>
+        </div>
+    </div>
+    </section>
+    <section class="bg-primary">
+        <div class="container">
+        <div class="split">
+            <div>
+                <img src="" alt="mountains with good looking sunset">
+            </div>
+            <div>
```

Comment:
please it is like you forgot to add in images . The image (src=" ") are empty please kindly consider adding in that missing source reference